.github/workflows/sync-linguist.yml

```diff
@@ -22,6 +22,7 @@ jobs:
         set -euo pipefail
         IFS=$'\n\t'
         commit=$(sed --quiet --regexp-extended 's/[[:space:]]+commit[[:space:]]+=[[:space:]]"([a-f0-9]{40})"/\1/p' internal/code-generator/generator/generator_test.go)
+        echo "Previous commit: ${commit::8}"
```

Comment:
I would move this to other PR as it's not directly related to this fix and should be discussed if actually needed

Reply:
Sure, I was initially trying to figure out why the sync-linguist action wasn't working and I thought that step of the process wasn't very expressive, but I've removed that change for now.